gateway-discovery-cm/src/test/java/org/apache/knox/gateway/topology/discovery/cm/ClouderaManagerServiceDiscoveryTest.java

```diff
@@ -562,6 +565,30 @@ public void testNiFiRegistryDiscoverySSL() {
     doTestNiFiRegistryDiscovery(true);
   }
 
+  @Test
+  public void testCMDiscovery() {
```

Comment:
I think it would be better to separate these, as has been done with other services (e.g., testCMDiscoveryAPI(), testCMDiscoveryUI())

Reply:
I see, sure, I'll update this.